OWNERS

```diff
@@ -2,11 +2,12 @@
 
 approvers:
   - codificat
+  - harshad16
   - fridex
+  - goern
   - kpostoffice
   - mayacostantini
   - sesheta
 reviewers:
   - codificat
   - mayacostantini
-  - harshad16
```

Comment:
sure - I actually did not mean to remove... I wrongly did a "cut" instead of a "copy" before pasting into approvers.

Fixed now, thanks for pointing out